website/docs/language/backend/azurerm.mdx

```diff
@@ -540,9 +540,11 @@ When authenticating using a Managed Identity (MSI) - the following fields are al
 
 When authenticating using a Service Principal with OpenID Connect (OIDC / Workload Identity Federation) - the following fields are also supported:
 
-* `oidc_request_url` - (Optional) The URL for the OIDC provider from which to request an ID token. This can also be sourced from the `ARM_OIDC_REQUEST_URL` or `ACTIONS_ID_TOKEN_REQUEST_URL` environment variables.
+* `ado_pipeline_service_connection_id` - (Optional) The Azure DevOps Pipeline Service Connection ID. This can also be sourced from the `ARM_ADO_PIPELINE_SERVICE_CONNECTION_ID` or `ARM_OIDC_AZURE_SERVICE_CONNECTION_ID` Environment Variables. The provider will look for values in this order and use the first it finds configured.
```

Comment:
I don't believe that note is necessary 
```suggestion
* `ado_pipeline_service_connection_id` - (Optional) The Azure DevOps Pipeline Service Connection ID. This can also be sourced from the `ARM_ADO_PIPELINE_SERVICE_CONNECTION_ID` or `ARM_OIDC_AZURE_SERVICE_CONNECTION_ID` Environment Variables.
```

Reply:
@jackofallops Asked to emphasize this in the PR to azurerm provider: https://github.com/hashicorp/terraform-provider-azurerm/pull/28674#discussion_r1944210746

Reply:
Understood! Thanks for the added context